modules/cluster-logging-vector-tech-preview.adoc

```diff
@@ -0,0 +1,116 @@
+// Module included in the following assemblies:
+//cluster-logging-release-notes.adoc
+
+:_content-type: PROCEDURE
+[id="cluster-logging-vector-tech-preview_{context}"]
+:FeatureName: Vector
+include::snippets/technology-preview.adoc[]
+
+[id="cluster-logging-about-vector"]
+= About Vector
+[role="_abstract"]
+Vector is an observability data pipeline currently offered as an alternative to the default collector for the {logging}.
+
+[IMPORTANT]
+====
+Vector does not support FIPS Enabled Clusters.
+====
+
+The following outputs are supported:
+
+--
+* `cloudwatch`. Amazon CloudWatch, a monitoring and log storage service hosted by Amazon Web Services (AWS).
+
+* `elasticsearch`. An external Elasticsearch instance. The `elasticsearch` output can use a TLS connection.
+
+* `fluentdForward`. An external log aggregation solution that supports Fluentd. This option uses the Fluentd *forward* protocols. The `fluentForward` output can use a TCP or TLS connection and supports shared-key authentication by providing a *shared_key* field in a secret. Shared-key authentication can be used with or without TLS.
+
+* `kafka`. A Kafka broker. The `kafka` output can use a TCP or TLS connection.
+
+* `loki`. Loki, a horizontally scalable, highly available, multi-tenant log aggregation system.
+
+* `syslog`. An external log aggregation solution that supports the syslog link:https://tools.ietf.org/html/rfc3164[RFC3164] or link:https://tools.ietf.org/html/rfc5424[RFC5424] protocols. The `syslog` output can use a UDP, TCP, or TLS connection.
+
+[id="cluster-logging-enabling-vector"]
+= Enabling Vector
+[role="_abstract"]
+Vector is not enabled by default. Use the following steps to enable Vector on your {product-title} cluster.
+
+[IMPORTANT]
+====
+Vector does not support FIPS Enabled Clusters.
+====
+
+.Prerequisites
+
+* {product-title}: 4.10
+* {logging-title-uc}: 5.4
+* FIPS disabled
+
+.Procedure
+
+. Edit the `ClusterLogging` custom resource (CR) in the `openshift-logging` project:
++
+[source,terminal]
+----
+$ oc -n openshift-logging edit ClusterLogging instance
+----
+
+. Add `logging.openshift.io/preview-vector-collector: enabled` to the `ClusterLogging` custom resource (CR).
```

Comment:
Please mention the keyword `annotation`
Add annotation `logging.openshift.io/preview-vector-collector: enabled` to the `ClusterLogging` custom resource (CR)